contracts/contracts/strategies/aerodrome/AerodromeAMOStrategy.sol

```diff
@@ -803,20 +830,12 @@ contract AerodromeAMOStrategy is InitializableAbstractStrategy {
     }
 
     /**
-     * @notice Withdraw an `amount` of assets from the platform and
-     *         send to the `_recipient`.
-     * @param _recipient  Address to which the asset should be sent
-     * @param _asset      WETH address
-     * @param _amount     Amount of WETH to withdraw
+     * @dev This function removes the appropriate amount of liquidity to assure that the required
+     * amount of WETH is available on the contract
+     *
+     * @param _amount  WETH balance required on the contract
      */
-    function withdraw(
-        address _recipient,
-        address _asset,
-        uint256 _amount
-    ) external override onlyVault nonReentrant {
-        require(_asset == WETH, "Unsupported asset");
-        require(_recipient == vaultAddress, "Only withdraw to vault allowed");
-
+    function _assureWETHBalance(uint256 _amount) internal {
```

Comment:
⚪️ `_ensureWETHBalance`.  "Assure" means "I'm trying to convince you things are good so that you feel better".  "Ensure" means "I will take the actions necessary to be sure that something is done.". We want the second.

Reply:
Given that this is called by the very active sounding `_removeLiquidityToFacilitateSwap ` and calls the active sounding `_removeLiquidity`, I might go with either more active name here. Maybe `_removeLiqudityToEnsureWETH`

Reply:
hah thanks yes ensure seems much more fitting. 
You assuring me ensure is the better word has me convinced :)